content/docs/command-reference/plots/show.md

```diff
@@ -492,3 +492,46 @@ file:///Users/usr/src/dvc_plots/index.html
 ```
 
 ![](/img/plots_show_confusion_normalized.svg)
+
```

Comment:
@tibor-mach something funny is happening here 🤔  Did you run the pre-commit hook?

Reply:
I didn't because the `.pre-commit-config.yaml` seems to be missing in the repo. There is a `.pre-commit-hooks.yaml` file but it is not a valid config...so I am bit confused about how I can run it.

Reply:
Yes, in this case it's installed if you run `yarn`.

Reply:
Here are a bit more details - https://dvc.org/doc/contributing/docs#development-environment, but you don't need to run everything. And we are figuring out how to simplify the workflow for the external contributors. We had an external tool (Restyled) that was taking care of running linters and creating a fixed PR on top of a contribution.

Reply:
I did run `yarn` though. I set up a locally hosted version of the website with `yarn develop` and had it opened as I was editing the code.

I now also ran all the liters etc. manually. Everything seems to be fine. Building it also works fine...

I do get the following warning and error when I run `yarn develop` though. Not sure if this might have something to do with the problem here (I am not exactly sure what the problem is as it works on my machine ... :-) )

```
(process:22910): GModule-CRITICAL **: 21:18:59.521: g_module_make_resident: assertion 'module != NULL' failed

(process:22910): VIPS-WARNING **: 21:18:59.521: unable to load "/usr/lib/vips-modules-8.13/vips-jxl.so" -- libjxl_threads.so.0.6: cannot open shared object file: No such file or directory
```